### DISCUSSION TITLE
Bosch BMCT-SLZ support as shutters control

### DISCUSSION THREAD

**Discussion by mmirannalti** [Koenkk/zigbee2mqtt#18754] [Devices]
Hi.
I am renovating my house and looking for a solution to shutters automation.
I am looking at Bosch BMCT-SLZ, I thought it was supported as according to documentation it exposed cover.
Reading the notes, it states "Roller shutter operation mode is not supported yet."

Can anyone help me clarifying this? I am planning to buy a number of them...

Thanks

**Comment by mmirannalti** [comment]
If anyone is reading this post, it is actually possible to use the BMCT-SLZ as shutters control.
It seems to be an issue with documentation.

**Reply by vertexxx** [reply]
Hey, 
thank you for the response!
How is your experience so far? Did you have any problems?

I am planning to use it for roller shutters (ger.: Rollladen) and blinds (ger.: Jalousie). According to Boschs instructional videos you are able to operate in two different modes, did you see any different modes exposed in z2m?

**Reply by mmirannalti** [reply]
Hi.
Sorry, I read your message only now.

I have around 10 BMCT-SLZ installed at home. I have been using them for 3 months with Rolladen, Jalousie and sun shades.
2 modes are exposed (shutter and switch).

Everything worked well so far. It is important to wire them correctly, the up & down contacts cannot be configured via software,   

I still have to understand if and how I can handle the tilt for the Jalousie. 
It seems to be possible with the Bosch hub/app, I could not figure it out yet.

**Reply by michaelmell** [reply]
Hi,

I have a questions, since you have experience with the Bosch BMCT-SLZ controlling roller shutters.
Do you know, if it is possible to start the calibration process using Z2M?
(See my question [here](https://github.com/Koenkk/zigbee2mqtt/discussions/29156).)

Thanks in advance!

**Reply by mmirannalti** [reply]
Hi. 
I have never seen such a functionality in z2m.
You can do the calibration manually, providing the time in seconds for opening and closing.

**Reply by DerDreschner** [reply]
@michaelmell : The BMCT-SLZ support an automatic calibration process, but it's not being implemented / exposed in Z2M. I don't have an BMCT-SLZ here right now and would need to lend a shutter motor to support that.

**Reply by mmirannalti** [reply]
@DerDreschner I have a spare BMCT-SLZ, I live pretty close to you (Mainz) and I would be happy to borrow it to you.
I cannot help with the motor, unless you don't want to come to my place to test it :-)

**Reply by DerDreschner** [reply]
@mmirannalti : Ohh, that's nice, thank you! Especially because it looks like you have all three kind of shutters (rolladen, jalousie and sun shades)? Send me an email to github-2017@dreschner.net that we can discuss that further. :smile:
